catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+spec:
+  type: library
+  lifecycle: production
+  owner: team-shared-services
+  system: prima.it
+metadata:
+  name: opentelemetry_absinthe
+  description: An OpenTelemetry instrumentation library for Absinthe
+  tags:
+  - elixir
+  links:
+  - url: https://hex.pm/packages/opentelemetry_absinthe
+    icon: link
+    title: hex page
+
+  - url: https://hexdocs.pm/opentelemetry_absinthe/api-reference.html
+    icon: link
+    title: official documentation
+
+  - url: https://github.com/primait/bridge.rs
```

Comment:
```suggestion
  - url: https://github.com/primait/opentelemetry_absinthe
```

Reply:
(per altro il link di github non server perché ce l'hai già out of the box su backstage :D)

Reply:
allora tanto vale toglierlo